packages/chrome/src/elements/header/HeaderItemWrapper.tsx

```diff
@@ -6,13 +6,14 @@
  */
 
 import React, { HTMLAttributes } from 'react';
-import { StyledHeaderItemWrapper } from '../../styled';
+import { StyledHeaderItemWrapper, IStyledBaseHeaderItemProps } from '../../styled';
```

Comment:
Moving to TS for the examples found we were missing some types for this component.